gradle/libs.versions.toml

```diff
@@ -29,6 +29,7 @@ kotlinx-html = "0.9.1"
 
 ## Markdown
 jetbrains-markdown = "0.5.2"
+fastutil = "8.5.12"
```

Comment:
just in case, I haven't mentioned it, but this is the same version, which is used in `jetbrains-markdown` right now

Reply:
Does this version need to be aligned with the version of `fastutil` from some other dependency, like the version used in `jetbrains-markdown` or in `kotlin-compiler`? 

If it needs to be aligned, I think there should definitely be a comment about it with a link, similar to how it's done for `intellij-platform` up above, so that we are not trigger-happy with updating it

Reply:
Overall, the version should be compatible with what is used in `kotlin-compiler` (AFAIK the same version is used in `IntelliJ` dependencies) and with `markdown-jb` library.
For now, `markdown-jb` uses newer version - so I would think we need to align with it.
I will write additional comment here